libsolidity/codegen/YulUtilFunctions.h

```diff
@@ -42,6 +43,7 @@ class ArrayType;
 class MappingType;
 class IntegerType;
 class StructType;
+class VariableDeclaration;
```

Comment:
1. ASTForward is even better, thanks for that.
2. Yes, all of the `...Type` forward declarations are unnecessary - I'd rather not get rid of them in this PR though if that's OK? I mean, I could, but not sure such a change belongs here.

Reply:
Ah, unfortunately `ASTForward.h` will not work here, since `VariableDeclaration` is an incomplete type (with nested `Location` enum). I can still just leave the `AST.h` include and remove the `VariableDeclaration` forward declaration.

Reply:
> I'd rather not get rid of them in this PR though if that's OK?

It's ok, though this seems simple enough that I don't see much point in starting a separate PR just for it. Putting it in a commit or its own is enough to cleanly separate it from other changes.